README.md

```diff
@@ -6,39 +6,68 @@ Tarantool documentation source, published at https://www.tarantool.io/doc/.
 
 ## How to build Tarantool documentation using [Docker](https://www.docker.com)
 
-### Build docker image
+### Prepare for work
 
-```bash
-docker build -t tarantool-doc-builder .
-```
+First of all, pull the image for building the docs.
 
-### Build Tarantool documentation using *tarantool-doc-builder* image
-## NOTE: 
-> Run this command only if you don't have untracked files!
-  check it by `git status` 
-  Also failures during git submodule update can be fixed by:
-   ```bash
-   git submodule deinit -f .
-   git submodule update --init
-   ```
-
-Init and update submodules:
-```bash
-docker run --rm -it -v $(pwd):/doc tarantool-doc-builder sh -c "./update_submodules.sh"
-```
-or do it manually:
 ```bash
-git submodule init
-git submodule update
-git pull --recurse-submodules
-git submodule update --remote --merge
+docker pull tarantool/doc-builder
 ```
 
-Init make commands:
+Next, initialize a Makefile for your OS:
+
 ```bash
-docker run --rm -it -v $(pwd):/doc tarantool-doc-builder sh -c "cmake ."
+docker run --rm -it -v $(pwd):/doc tarantool/doc-builder sh -c "cmake ."
 ```
 
+### Update submodules and generate documentation sources from code
+
+A big part of documentation sources comes from several other projects,
+connected as Git submodules.
+To include their latest contents in the docs, run these two steps.
+
+1.  Update the submodules:
+
+    ```bash
+    docker run --rm -it -v $(pwd):/doc tarantool/doc-builder sh -c "./pull_submodules.sh"
```

Comment:
why do you use direct command instead of the new `make pull-submodules`?

Reply:
@artembo fixed!